packages/graphql-transformer-core/src/collectDirectives.ts

```diff
@@ -65,7 +65,15 @@ export function collectDirectivesByType(sdl: string): Object {
   if (!sdl) {
     return {};
   }
-  const doc = parse(sdl);
+  let doc;
+  try {
+    doc = parse(sdl);
+  } catch (e) {
+    if (e.message.includes('Syntax Error')) {
```

Comment:
Technically, `e` could be anything, so we should probably use optional chaining here.